rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt

```diff
@@ -0,0 +1,53 @@
+package com.badoo.reaktive.rxjavainterop
```

Comment:
probably we should also add interop with rxjava schedulers to reuse the thread pools

Reply:
Can you tell any use cases?

Reply:
currently, we reuse rxjava 1/2 schedulers so we don't have duplicated thread pools

Reply:
Sounds reasonable, would it be enough to just convert rxjava2 schedulers to Reaktive. So Reaktive could reuse rxjava2's schedules.

Reply:
I have pushed the changes, could you please have a look?